client/src/review-components/ReviewForm.jsx

```diff
@@ -1,76 +1,145 @@
-import React from 'react';
+import React, {useState} from 'react';
 import ReactDOM from 'react-dom/client';
-import Stars from './Stars.jsx';
+import HoverStars from './HoverStars.jsx';
+import axios from 'axios';
 
 function ReviewForm(props) {
+
+  let charDetails = {
```

Comment:
As a level up, this hard-coded business logic is best kept out of a code file and in a data file. Like a fixture, most easily a JSON file (tho there are tradeoffs/discussions) as to file types.

In another file at least you can signal knowledge that these values would be best stored in a CMS (content management system)